evap/staff/views.py

```diff
@@ -1581,50 +1583,50 @@ def questionnaire_create(request):
     return render(request, "staff_questionnaire_form.html", dict(form=form, formset=formset, editable=True))
 
 
+def disable_all_except_named(fields: Dict[str, Any], names_of_editable: List[str]):
```

Comment:
Did we discuss type annotations before? I feel like we did but I can't remember

Reply:
`names_of_editable` just has to be some kind of collection that can be tested with `in` / `__contains__()`, so it should probably be [`Container[str]`](https://docs.python.org/3/library/collections.abc.html#collections.abc.Container). The call could be made with a tuple or a set instead of a list.